starter-code/database.js

```diff
@@ -98,11 +98,10 @@ class Database {
       if (error){
         callback(error);
       } else {
-        // LAB 5
-        // Implement the query to list all products
-        // remeber once it's finish to comment callback('Error listing products');
+       
+       database.collection('products').find(callback);
```

Comment:
you need a name for the products as well
        **database.collection('products').find(products.productName, callback);**